packages/api/test/utils/checkAgainstSpec.ts

```diff
@@ -16,7 +16,7 @@ ajv.addKeyword({
   errors: false,
 });
 
-ajv.addFormat("hex", /^0x[a-fA-F0-9]+$/);
+ajv.addFormat("hex", /^0x[a-fA-F0-9]*$/);
```

Comment:
this is required as `extraData.defaultValue()` produces `"0x"` which should be considered a valid hex value, this also matches the [regex in the spec](https://github.com/ethereum/beacon-APIs/blob/02d595daf39943430f54aa978f88a5c50d957e64/types/primitive.yaml#L118)

otherwise we will get the following error when running spec tests
```
/body/data/execution_payload/extra_data - must match format "hex"
```